src/libsystemd/sd-event/sd-event.c

```diff
@@ -126,7 +126,7 @@ struct sd_event {
         LIST_HEAD(struct inode_data, inode_data_to_close);
 
         /* A list of inotify objects that already have events buffered which aren't processed yet */
-        LIST_HEAD(struct inotify_data, inotify_data_buffered);
+        LIST_HEAD(struct inotify_data, buffered_inotify_data_objects);
```

Comment:
"objects"? i am not sure i would call this "objects". they are dead structs, not more...

this renaming smells a bit like bike shedding to me though, but I don't mind, if this is important to you.

Reply:
I changed the name again, to `buffered_inotify_data_list`.

I don't particularly *like* the verbose name, but @yuwata requested a change, and I think it makes sense to have the general rule that container-like fields have a name that indicates that. This makes it easier for the reader to avoid a mistake where they forget that the thing is just one-of-many. If you don't like the new name, I can drop the commit, but I think it's better to rename.